src/content/en/tools/lighthouse/_toc.yaml

```diff
@@ -154,6 +154,8 @@ toc:
     path: /web/tools/lighthouse/audits/description
   - title: Document Doesn't Have A Title Element
     path: /web/tools/lighthouse/audits/title
+  - title: Document doesn't have a valid rel=canonical
```

Comment:
I'll eventually update all the titles to match the Lighthouse report formatting.